### DISCUSSION TITLE
Is there a way to make the logs stored in SO Immutable?

### DISCUSSION THREAD

**Discussion by Phillip-Samson** [Security-Onion-Solutions/securityonion#10591] [General]
SO 2.3 latest and 2.4 Beta 3

**Comment by dougburks** [comment]
Logs are stored in Elasticsearch and you should be able to manually set an Elasticsearch index to read-only. I haven't specifically tested this, so you would want to test it thoroughly to make sure it doesn't interfere with our automatic disk cleanup.
